### PR TITLE
Rearrange test projects and make them support partial build

### PR DESCRIPTION
**Pull request by sergeybykov**

### CODE REVIEW

**Comment by sergeybykov** [comment]
@dotnet-bot test bvt

**Comment by ReubenBond** [comment]
@dotnet-bot test BVT

File: test/DependencyInjection.Tests/Directory.Build.props [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+  </PropertyGroup>
+
+  <Import Project="$(_ParentDirectoryBuildPropsPath)" Condition="Exists('$(_ParentDirectoryBuildPropsPath)')"/>
+
+  <PropertyGroup Condition="$(OrleansCoreVersion)!=$(VersionPrefix)">
+    <IsPackable>false</IsPackable>
+  </PropertyGroup>
+
+  <PropertyGroup>
+    <GenerateDocumentationFile>true</GenerateDocumentationFile>
```
Comment (ReubenBond) [inline test/DependencyInjection.Tests/Directory.Build.props]: false?

**Comment by ReubenBond** [comment]
Typo on TransactionManagerBentchmarks.cs

**Pull request merged**